packages/block-library/src/post-author/edit.js

```diff
@@ -114,84 +122,152 @@ function PostAuthorEdit( {
+								spacing={ 4 }
+								style={ { gridColumn: '1 / -1' } }
+							>
+								{ ( showCombobox && (
+									<ComboboxControl
+										__next40pxDefaultSize
+										__nextHasNoMarginBottom
+										label={ __( 'Author' ) }
+										options={ authorOptions }
+										value={ authorId }
+										onChange={ handleSelect }
+										allowReset={ false }
+									/>
+								) ) || (
+									<SelectControl
+										__next40pxDefaultSize
+										__nextHasNoMarginBottom
+										label={ __( 'Author' ) }
+										value={ authorId }
+										options={ authorOptions }
+										onChange={ handleSelect }
+									/>
+								) }
+							</VStack>
+						) }
+						<ToolsPanelItem
 							label={ __( 'Show avatar' ) }
-							checked={ showAvatar }
-							onChange={ () =>
-								setAttributes( { showAvatar: ! showAvatar } )
+							isShownByDefault
+							hasValue={ () => showAvatar !== true }
+							onDeselect={ () =>
+								setAttributes( { showAvatar: true } )
 							}
-						/>
-						{ showAvatar && (
-							<SelectControl
-								__next40pxDefaultSize
+						>
+							<ToggleControl
 								__nextHasNoMarginBottom
-								label={ __( 'Avatar size' ) }
-								value={ attributes.avatarSize }
-								options={ avatarSizes }
-								onChange={ ( size ) => {
+								label={ __( 'Show avatar' ) }
+								checked={ showAvatar }
+								onChange={ () =>
 									setAttributes( {
-										avatarSize: Number( size ),
-									} );
-								} }
+										showAvatar: ! showAvatar,
+									} )
+								}
 							/>
+						</ToolsPanelItem>
+						{ showAvatar && (
+							<ToolsPanelItem
+								label={ __( 'Avatar size' ) }
+								isShownByDefault
+								hasValue={ () => attributes.avatarSize !== 48 }
+								onDeselect={ () =>
+									setAttributes( { avatarSize: 48 } )
+								}
+							>
+								<SelectControl
+									__next40pxDefaultSize
+									__nextHasNoMarginBottom
+									label={ __( 'Avatar size' ) }
+									value={ attributes.avatarSize }
+									options={ avatarSizes }
+									onChange={ ( size ) => {
+										setAttributes( {
+											avatarSize: Number( size ),
+										} );
+									} }
+								/>
+							</ToolsPanelItem>
 						) }
-						<ToggleControl
-							__nextHasNoMarginBottom
+						<ToolsPanelItem
 							label={ __( 'Show bio' ) }
-							checked={ showBio }
-							onChange={ () =>
-								setAttributes( { showBio: ! showBio } )
+							isShownByDefault
+							hasValue={ () => !! showBio !== false }
+							onDeselect={ () =>
+								setAttributes( { showBio: undefined } )
 							}
-						/>
-						<ToggleControl
-							__nextHasNoMarginBottom
-							label={ __( 'Link author name to author page' ) }
-							checked={ isLink }
-							onChange={ () =>
-								setAttributes( { isLink: ! isLink } )
+						>
+							<ToggleControl
+								__nextHasNoMarginBottom
+								label={ __( 'Show bio' ) }
+								checked={ showBio }
```

Comment:
```suggestion
								checked={ !! showBio }
```

Maybe not the role of this PR, but it avoids the browser warning when enabling the toggle:

> Warning: A component is changing an uncontrolled input to be controlled. This is likely caused by the value changing from undefined to a defined value, which should not happen. Decide between using a controlled or uncontrolled input element for the lifetime of the component.